provision/terraform/reporting/nifi-registry/terraform.tfvars

```diff
@@ -0,0 +1,16 @@
+nr-instance-type = "t2.medium"
+nr-volume-size = "200"
+nr-ssh-key = "ona-terraform-tbd"
```

Comment:
Could we make sure that this AWS key is in our S3 bucket?

Reply:
Done. I've changed the key to `openlmis-reporting`. I've copied the private key to the S3 bucket.